src/components/DefaultRiotServer.js

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2016 OpenMarket Ltd
```

Comment:
this looks wrong

Reply:
Copypasta strikes again. This is true for all files; should it be `Copyright 2019 Matrix.org Foundation C.I.C.` as I've seen elsewhere?